lib/parallel_tests/cli.rb

```diff
@@ -291,9 +312,14 @@ def execute_shell_command_in_parallel(command, num_processes, options)
       abort if results.any? { |r| r[:exit_status] != 0 }
     end
 
-    def report_time_taken
+    def report_time_taken(consolidated_results)
       seconds = ParallelTests.delta { yield }.to_i
-      puts "\nTook #{seconds} seconds#{detailed_duration(seconds)}"
+      message = "Took #{seconds} seconds#{detailed_duration(seconds)}"
+      consolidated_results[:time_taken] = {
```

Comment:
how long the command took is easily gathered by whatever process runs the parallel_tests command
also `time_taken: seconds` might be good enough, don't really need the custom format if this goes through post-processing anyway